packages/ui/README.md

```diff
@@ -0,0 +1,56 @@
+# `@prairielearn/ui`
+
+UI components shared between PrairieLearn and PrairieTest.
+
+TODO: how to share CSS?
```

Comment:
We don't have a good story for this yet -- should we distribute CSS snippets alongside the components? We have `apps/prairielearn/assets/stylesheets/tanstackTable.css` with:

```
body.no-user-select {
  user-select: none;
  -webkit-user-select: none;
}
```

Reply:
Sounds good, thanks!

Reply:
Fixed in [089d968](https://github.com/PrairieLearn/PrairieLearn/pull/13013/commits/089d968bc81610468d14287dbcb1a3f131644c39).